### ISSUE TITLE
I thought I found a bug - I was wrong

### ISSUE THREAD

**Issue by BigAlanM** [toolbox4minecraft/amidst#100]
This MAY be a problem with my machine, but when I do a Ctrl-C in either Amidst4.0b10 or AmidstExporter144 (the latest version which has much of the beta10 code) both programs place the displayed seed into the clipboard as indicated by a program called Free Clipboard Viewer.
HOWEVER, I can't paste the seed into Minecraft !!???!!.
To get Ctrl-V to paste the seed to MC I have to paste it to another program and then use Ctrl-C again, THEN Ctrl-V works in the Minecraft create new world dialog.
I've successfully done this by using Ctrl-C from the clipboard viewer display and by also doing a Ctrl-V to Notepad++ and then a Ctrl-C from there.
Even doing a Ctrl-V to the Chrome URL input and then Ctrl-V from there allows the seed to be pasted to MC.

Is it POSSIBLE that some windows clipboard flag isn't being set properly by the new versions?

**Comment by BigAlanM** [comment]
Upon further investigation I've found that yet again I'm getting my exercise by jumping to conclusions.
I downloaded Amidst beta 9 and then beta 8 and tried them.  Same results and I KNOW for sure that Ctrl-C worked in them before.
So on a whim I closed down Minecraft and the launcher and restarted them.
And, OF COURSE, Ctrl-C from all four Amidst programs can pasted to the newly started instance of Minecraft.
I surprised that I can walk at all as many times I've shot myself in the foot.  SIGH!
I'm not going to close this issue to hide my shame.
Let this be a lesson to everyone to do your research before shouting BUG!.
